pypeit/pypeit.py

```diff
@@ -295,33 +330,48 @@ def reduce_exposure(self, frames, bg_frames=[], std_outfile=None):
         Args:
             frame (:obj:`int`):
                 0-indexed row in :attr:`fitstbl` with the frame to
-                reduce
-            bgframes (:obj:`list`, optional):
-                List of frame indices for the background
+                reduce.
+            bg_frames (:obj:`list`, optional):
+                List of frame indices for the background.
             std_outfile (:obj:`str`, optional):
-                the name of a file with a previously PypeIt-reduced standard spectrum.
+                File with a previously reduced standard spectrum from
+                PypeIt.
 
         Returns:
-            dict: The dictionary containing the primary outputs of extraction
+            dict: The dictionary containing the primary outputs of
+            extraction.
+
         """
+
+        # TODO:
+        # - bg_frames should be None by default
+        # - change doc string to reflect that more than one frame can be
+        #   provided
+
         # if show is set, clear the ginga channels at the start of each new sci_ID
         if self.show:
+            # TODO: Put this in a try/except block?
             ginga.clear_all()
 
-        # Save the frame
+        # Save the frames
+        # TODO: Should these be set to self?  They're not used elsewhere
+        # in the class or in run pypeit.
         self.frames = frames
         self.bg_frames = bg_frames
         # Is this an IR reduction?
+        # TODO: Why specific to IR?
         self.ir_redux = True if len(bg_frames) > 0 else False
 
-        # JFH Why does this need to be ordered?
+        # TODO: JFH Why does this need to be ordered?
         sci_dict = OrderedDict()  # This needs to be ordered
         sci_dict['meta'] = {}
         sci_dict['meta']['vel_corr'] = 0.
         sci_dict['meta']['ir_redux'] = self.ir_redux
 
         # Print status message
         msgs_string = 'Reducing target {:s}'.format(self.fitstbl['target'][self.frames[0]]) + msgs.newline()
+        # TODO: Print these when the frames are actually combined,
```

Comment:
Yes I agree that would probably make for cleaner code.

Reply:
Sounds good.  Leaving this as a TODO for now.